SimpleBrowser.UnitTests/OfflineTests/WindowsAndFrames.cs

```diff
@@ -1,154 +1,172 @@
-		}
-
-
-		[Test]
-		public void Accessing_New_Windows_Using_Event()
-		{
-            Browser b = new Browser(Helper.GetMoviesRequestMocker());
-			Browser newlyOpened = null;
-			b.NewWindowOpened += (b1, b2) =>
-			{
-				newlyOpened = b2;
-			};
-			b.Navigate("http://localhost/movies/");
-			Assert.That(b.Url == new Uri("http://localhost/movies/"));
-			var link = b.Find(ElementType.Anchor, FindBy.Text, "Details");
-			b.KeyState = KeyStateOption.Ctrl;
-			link.Click();
-			Assert.That(b.Windows.Count() == 2);
-			Assert.NotNull(newlyOpened);
-			Assert.That(b.Url.ToString() == "http://localhost/movies/");
-			Assert.That(newlyOpened.Url.ToString() == "http://localhost/movies/Movies/Details/1");
-		}
-
-
-		[Test]
-		public void ClosingBrowsers()
-		{
-			Browser b = new Browser(Helper.GetMoviesRequestMocker());
-			HttpRequestLog lastRequest = null;
-			b.RequestLogged += (br, l) =>
-			{
-				lastRequest = l;
-			};
-			b.Navigate("http://localhost/movies/");
-			Assert.That(b.Url == new Uri("http://localhost/movies/"));
-			var link = b.Find(ElementType.Anchor, FindBy.Text, "About us");
-			link.Click();
-			Assert.That(b.Url == new Uri("http://localhost/movies/"));
-			Assert.That(b.Windows.Count() == 2);
-			b.Close();
-			Assert.That(b.Windows.Count() == 1);
-			b.Windows.First().Close();
-			Assert.That(b.Windows.Count() == 0);
-			Assert.Throws(typeof(ObjectDisposedException), () => { Uri s = b.Url; });
-		}
-		[Test]
-		public void Page_With_IFrames()
-		{
-			Browser b = new Browser(Helper.GetFramesMock());
-			HttpRequestLog lastRequest = null;
-			b.RequestLogged += (br, l) =>
-			{
-				lastRequest = l;
-			};
-			b.Navigate("http://localhost/");
-			Assert.That(b.Frames.Count() == 2);
-
-			// now navigate away to a page without frames
-			b.Navigate("http://localhost/bla");
-			Assert.That(b.Frames.Count() == 0);
-			Assert.That(b.Windows.Count() == 1);
-		}
-		[Test]
-		public void GetAttribute_Backdoor_FrameHandle()
-		{
-			Browser b = new Browser(Helper.GetFramesMock());
-			HttpRequestLog lastRequest = null;
-			b.RequestLogged += (br, l) =>
-			{
-				lastRequest = l;
-			};
-			b.Navigate("http://localhost/");
-			var elm = b.Select("iframe");
-			string handle = elm.GetAttribute("SimpleBrowser.WebDriver:frameWindowHandle");
-			Assert.AreEqual(handle, "frame1");
-		}	
-		[Test]
-		public void Navigating_IFrames_Using_Target()
-		{
-			Browser b = new Browser(Helper.GetFramesMock());
-			HttpRequestLog lastRequest = null;
-			b.RequestLogged += (br, l) =>
-			{
-				lastRequest = l;
-			};
-			b.Navigate("http://localhost/");
-			Assert.That(b.Frames.Count() == 2);
-			Assert.That(b.Frames.First().Url == new Uri("http://localhost/subdirectory/frame.htm"));
-
-			b.Find("framelink").Click();
-			Assert.That(b.Frames.Count() == 2);
-			Assert.That(b.Url == new Uri("http://localhost/"));
-			Assert.That(b.Frames.First().Url == new Uri("http://localhost/bla.htm"));
-
-		}
-	}
-}
+﻿using System;

```

Comment:
Yes, the bottom two tests are new. I guess this happened because I edited on my mac. Whitespace/line ending issues.